### ISSUE TITLE
Adding SVG support?

### ISSUE THREAD

**Issue by BoomerScratch** [JeffaloBob/scratch-image-uploader#21]
Currently, if you upload an SVG image, the image will not appear and a 500 error occurs.
We should convert the SVG to PNG. This is easy: you can create a canvas element, render the SVG on it, and finally grab a picture of the canvas using canvas.toDataURL.
The harder part is that it doesn't work if the canvas is offscreen. To fix that, we can use @canvg. Create a file called canvg.js with [these](https://unpkg.com/canvg@latest/lib/umd.js) contents, and also add it to the content scripts in manifest.json.
We first need to merge #20 before I can start working on it.

**Comment by BoomerScratch** [comment]
Okay, this is kinda hard to do 😅

**Comment by easrng** [comment]
or, y'know, `OffscreenCanvas`

**Comment by BoomerScratch** [comment]
@easrng Yes, that is what I tried. I noticed some days ago that canvg isn't needed; it's built-in in most browsers.
